app/services/calculate_candidate.rb

```diff
@@ -0,0 +1,14 @@
+class CalculateCandidate
+  def initialize(answers)
+    @answers = answers
+  end
+
+  def execute
+    answers_by_candidates = @answers.group_by(&:candidate)
+    max = (answers_by_candidates.map { |k, v| v.count }).max
+    (answers_by_candidates.map { |k, v| (v.count == max) ? 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.<br>Trailing whitespace detected.